infoschema/tables.go

```diff
@@ -649,7 +649,7 @@ func dataForProcesslist(ctx sessionctx.Context) [][]types.Datum {
 			pi.User,
 			pi.Host,
 			pi.DB,
-			pi.Command,
+			mysql.Command2Str[pi.Command],
```

Comment:
There are multiple place that build process information, could you abstraction a function ?

Reply:
Done